helpo-api/requirements.txt

```diff
@@ -4,7 +4,7 @@ fixmydjango==0.3.2
 django==1.11.15
 celery==4.2.1
 redis==2.10.6
-django-js-reverse==0.8.2
+django-js-reverse===0.9.1
```

Comment:
Ahora te pusheo las `latest` tanto de api como de web con estos cambios y con pandas just in case

Reply:
Dale